### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [JovanaPerisic/TeaCozy#1]
When including images, it is best practice always to add the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute). The reason for this is because the alt attribute is used for both accessibility and as a backup, in case your image cannot be rendered. 

Here is a [gist](https://gist.github.com/jim-rosales/2d3d6e04d880b8365cc7933009106446) that showcases the use of the alt attribute on all the images.

https://github.com/JovanaPerisic/TeaCozy/blob/3ea66c36cdb4793e1abf81a73aff6d7535e544fe/index.html#L10
